### ISSUE TITLE
Standalone "Shipments" to create quick shipment sheets not linked to any order

### ISSUE THREAD

**Issue by relaxibus** [Dolibarr/dolibarr#22943]
### Feature Request

A Standalone "Shipments" to create quick shipment sheets not linked to any order is missing. 

### Use case

It is in the nature of business that we need to ship stuff not related to any order. Now we need to create a shipment sheet using Word or similar. This creates an overhead as Dolibarr users would like to use Dolibarr as the only tool.

### Suggested implementation

From the "Shipments" menu add "Quick Shipments (or similar)" menu

**Comment by edupulpillo** [comment]
Really will be useful. A lot of times only shipments are triggered directly with invoices and no linked to any order.

**Comment by relaxibus** [comment]
Any update from the developers? Will you add this feature?

**Comment by diplaros** [comment]
I also think that this is usefull feature.

**Comment by relaxibus** [comment]
yeah, don't understand why they take so long to add such a simple but useful feature. Ridiculous!

**Comment by josett225** [comment]
Still required

**Comment by edupulpillo** [comment]
Still required
